src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/InteractiveRequest.cs

```diff
@@ -179,5 +179,11 @@ private async Task<MsalTokenResponse> RunBrokerWithInstallUriAsync(string broker
             Metrics.IncrementTotalAccessTokensFromBroker();
             return tokenResponse;
         }
+
+        protected override bool ShouldNotCacheTokenResponse()
```

Comment:
AFAIK all request support POP. We should not cache pop tokens. So I don't understand why we'd need a virtual method in RequestBase which some requests can override.